src/lib/isCreditCard.js

```diff
@@ -1,7 +1,7 @@
 import assertString from './util/assertString';
 
 /* eslint-disable max-len */
-const creditCard = /^(?:4[0-9]{12}(?:[0-9]{3,6})?|5[1-5][0-9]{14}|(222[1-9]|22[3-9][0-9]|2[3-6][0-9]{2}|27[01][0-9]|2720)[0-9]{12}|6(?:011|5[0-9][0-9])[0-9]{12,15}|3[47][0-9]{13}|3(?:0[0-5]|[68][0-9])[0-9]{11}|(?:2131|1800|35\d{3})\d{11}|6[27][0-9]{14})$/;
+const creditCard = /^(?:4[0-9]{12}(?:[0-9]{3,6})?|5[1-5][0-9]{14}|(222[1-9]|22[3-9][0-9]|2[3-6][0-9]{2}|27[01][0-9]|2720)[0-9]{12}|6(?:011|5[0-9][0-9])[0-9]{12,15}|3[47][0-9]{13}|3(?:0[0-5]|[68][0-9])[0-9]{11}|(?:2131|1800|35\d{3})\d{11}|6[27][0-9]{14}|((5018|5020|5038|6304|6759|6761|6763)[0-9]{8,15})|(63[7-9][0-9]{13}))$/;
```

Comment:
(not related to this PR)
Haven't run any test but this regex smells like vulnerable to redos attacks.

Reply:
@fedeci Could you please test this PR?

Reply:
You should add tests in the proper file.

Reply:
Test is already passed, you can review this.

> # [Codecov](https://codecov.io/gh/validatorjs/validator.js/pull/1701?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs) Report
> > Merging [#1701](https://codecov.io/gh/validatorjs/validator.js/pull/1701?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs) ([90599bc](https://github.com/validatorjs/validator.js/commit/90599bc2cd60db88b17406173aab048344e6f679)) into [master](https://codecov.io/gh/validatorjs/validator.js/commit/8c4b3b35c3bd36dc3983aadb045422847154451b?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs) ([8c4b3b3](https://github.com/validatorjs/validator.js/commit/8c4b3b35c3bd36dc3983aadb045422847154451b)) will **not change** coverage.
> > The diff coverage is `100.00%`.
> 
> [![Impacted file tree graph](https://camo.githubusercontent.com/2f5a7677f3ba8bd66ea75f17d005f93e95e6d960e618137285953852ef71c29c/68747470733a2f2f636f6465636f762e696f2f67682f76616c696461746f726a732f76616c696461746f722e6a732f70756c6c2f313730312f6772617068732f747265652e7376673f77696474683d363530266865696768743d313530267372633d707226746f6b656e3d597a52336b624e316d322675746d5f6d656469756d3d726566657272616c2675746d5f736f757263653d6769746875622675746d5f636f6e74656e743d636f6d6d656e742675746d5f63616d706169676e3d70722b636f6d6d656e74732675746d5f7465726d3d76616c696461746f726a73)](https://codecov.io/gh/validatorjs/validator.js/pull/1701?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs)
> 
> ```diff
> @@            Coverage Diff            @@
> ##            master     #1701   +/-   ##
> =========================================
>   Coverage   100.00%   100.00%           
> =========================================
>   Files          101       101           
>   Lines         2005      2005           
>   Branches       452       452           
> =========================================
>   Hits          2005      2005           
> ```
> 
> [Impacted Files](https://codecov.io/gh/validatorjs/validator.js/pull/1701?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs)	Coverage Δ	
> [src/lib/isCreditCard.js](https://codecov.io/gh/validatorjs/validator.js/pull/1701/diff?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs#diff-c3JjL2xpYi9pc0NyZWRpdENhcmQuanM=)	`100.00% <100.00%> (ø)`	
> [Continue to review full report at Codecov](https://codecov.io/gh/validatorjs/validator.js/pull/1701?src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs).
> 
> > **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs)
> > `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> > Powered by [Codecov](https://codecov.io/gh/validatorjs/validator.js/pull/1701?src=pr&el=footer&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs). Last update [8c4b3b3...90599bc](https://codecov.io/gh/validatorjs/validator.js/pull/1701?src=pr&el=lastupdated&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=validatorjs).

Reply:
We should add both successful and failing test cases for the new features introduced in the PR even if the tests are passing